docs/en/operations/table_engines/file.md

```diff
@@ -14,7 +15,10 @@ Usage examples:
 File(Format)
 ```
 
-`Format` should be supported for either `INSERT` and `SELECT`. For the full list of supported formats see [Formats](../../interfaces/formats.md#formats).
+The `Format` parameter specifies one of the available file formats. If the
+format supports input, `SELECT` queries can be performed, and if it supports
```

Comment:
sounds like word-by-word translation from Russian... something like this would be better I think:

To perform `SELECT` the chosen format must support input and to perform `INSERTz it must support output.